software/bewcloud.yml

```diff
@@ -0,0 +1,18 @@
+name: "bewCloud"
+website_url: "https://bewcloud.com"
+source_code_url: "https://github.com/bewcloud/bewcloud"
+description: "A simpler alternative to Nextcloud and ownCloud's RSS Reader, File Sharing + Sync, Notes, and Photos."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Deno
+  - Nodejs
+  - Docker
+tags:
+  - File Transfer & Synchronization
+  - Groupware
+  - Feed Readers
+  - Knowledge Management Tools
```

Comment:
```suggestion
```

does not apply, if this refers to the notes functionality, this is covered by `Note-taking & Editors`